tests/benchmark/stateful/bloatnet/depth_benchmarks/test_worst_case_attack_execute.py

```diff
@@ -0,0 +1,539 @@
+
+This test implements a worst-case scenario for Ethereum block processing that exploits
+the computational complexity of Patricia Merkle Trie operations. It uses CREATE2 to deploy
+contracts at pre-mined addresses with shared prefixes, maximizing trie traversal depth.
+
+Key features:
+- Deploys 15,000 contracts via CREATE2 to addresses with configurable prefix sharing depth
+- Each contract has deep storage slots with configurable trie depth
+- Executes optimized attack bytecode that performs multiple SSTORE operations
+- Respects EIP-170 (24KB contract size) and Fusaka (16M gas per tx) limits
+- Verifies that deep storage slots are correctly modified
+
+Test parameters:
+- storage_depth: Depth of storage slots in the contract (e.g., 10)
+- account_depth: Depth of account address prefix sharing (e.g., 5)
+- Fixed 15,000 contracts deployed/attacked
+- Gas per attack call: 50,000 (sufficient for cold SSTORE operations)
+- Compilation requires: solc --metadata-hash none (for reproducible bytecode)
+"""
+
+import json
+import pytest
+from pathlib import Path
+from subprocess import run, PIPE
+from execution_testing import (
+    Account,
+    Alloc,
+    Block,
+    BlockchainTestFiller,
+    Bytecode,
+    Fork,
+    Op,
+    Transaction,
+    Address,
+)
+
+# Attack function selector for WorstCaseERC20.attack(uint256)
+ATTACK_SELECTOR = 0x64dd891a  # attack(uint256) - verified with: cast sig "attack(uint256)"
+
+# Maximum gas per transaction (Fusaka EIP limit)
+MAX_GAS_PER_TX = 16_000_000
+
+NUM_CONTRACTS = 15000  # Full test with 15,000 contracts
+
+# Nick's deterministic deployer address (must be pre-deployed in execute mode)
+NICK_DEPLOYER = Address("0x4e59b44847b379578588920ca78fbf26c0b4956c")
+
+
+def load_create2_data(storage_depth, account_depth):
+    """
+    Load the pre-mined CREATE2 addresses for given depth parameters.
+
+    Args:
+        storage_depth: Depth of storage slots in the contract (e.g., 10)
+        account_depth: Depth of account address prefix sharing (e.g., 5)
+
+    Returns dict with:
+        - init_code_hash: Expected hash for reproducible compilation
+        - contracts: List of dicts with 'salt' and 'auxiliary_accounts'
+    """
+    json_filename = f"s{storage_depth}_acc{account_depth}.json"
+    json_path = Path(__file__).parent / json_filename
+
+    if not json_path.exists():
+        raise FileNotFoundError(f"Pre-mined data not found: {json_filename}")
+
+    with open(json_path, 'r') as f:
+        return json.load(f)
+
+
+def compile_attack_orchestrator():
+    """
+    Compile the AttackOrchestrator.sol contract.
+
+    Returns:
+        bytes: Deployment bytecode for the orchestrator contract
+    """
+    sol_path = Path(__file__).parent / "AttackOrchestrator.sol"
+
+    if not sol_path.exists():
+        raise FileNotFoundError(f"AttackOrchestrator.sol not found at {sol_path}")
+
+    # Compile with optimization and no metadata for reproducibility
+    result = run(
+        [
+            "solc",
```

Comment:
To fully reproduce would also need to pin a solc version, as different versions can compile to different results.